### DISCUSSION TITLE
Memory consumption is not displayed

### DISCUSSION THREAD

**Discussion by siliciumix** [immich-app/immich#11574] [Q&A]
Hello,

I have installed version 1.111.0 in a Docker container. The data (library and PostgreSQL) is stored on a Samba share. Everything is working very well. However, in the sidebar at the bottom, where the used and maximum storage is displayed, I always see "112 KiB of 2.9 TiB." The 2.9 TiB is correct, but the 112 KiB can't be right. In the server statistics, I see that I have 14,763 photos and 438 videos, with a storage usage of 82 GiB.

The 112 KiB value doesn't change even when I upload new images. Even clearing the browser data (cache, etc.) doesn't help.

What else do I need to adjust to get the correct storage usage displayed?

Thank you,
Michael

**Comment by bo0tzz** [comment]
You already asked this in  #11562
